mm2src/crypto/Cargo.toml

```diff
@@ -7,17 +7,24 @@ edition = "2018"
 doctest = false
 
 [dependencies]
+aes = "0.8.3"
+argon2 = { version = "0.5.2", features = ["zeroize"] }
 arrayref = "0.3"
 async-trait = "0.1"
+base64 = "0.11.0"
 bip32 = { version = "0.2.2", default-features = false, features = ["alloc", "secp256k1-ffi"] }
+bip39 = { version = "2.0.0", features = ["rand_core", "zeroize"], default-features = false }
 bitcrypto = { path = "../mm2_bitcoin/crypto" }
 bs58 = "0.4.0"
+cbc = "0.1.2"
+cipher = "0.4.4"
 common = { path = "../common" }
 derive_more = "0.99"
 enum_from = { path = "../derives/enum_from" }
 enum-primitive-derive = "0.2"
 futures = "0.3"
 hex = "0.4.2"
+hmac = "0.12.1"
```

Comment:
We already have `0.11.0` in our dependency tree.

Reply:
We already had `0.12.1` in our dependency tree too before this PR.

Reply:
My bad